ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -990,11 +990,14 @@ Rectangle {
         width: messageInput.width
         filter: messageInputField.text
         cursorPosition: messageInputField.cursorPosition
-        property: ["name", "nickname", "ensName", "alias"]
+        property: ["nickname", "ensName", "name", "alias"]
         inputField: messageInputField
         onItemSelected: function (item, lastAtPosition, lastCursorPosition) {
             messageInputField.forceActiveFocus();
-            let name = item.name.replace("@", "")
+            const name = ProfileUtils.displayName(item.nickname.replace("@", ""),
+                                                  item.ensName.replace("@", ""),
+                                                  item.name.replace("@", ""),
+                                                  item.alias.replace("@", ""))
```

Comment:
You could use a `find` on the `item` just like we do in `SuggestionBoxPanel`

```qml
item[suggestionsBox.property.find(p => !!item[p])].replace("@", "")
```

Reply:
Done, thanks for the hint!